embedding-sdk/unauthenticated/index.html

```diff
@@ -0,0 +1,81 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Charts Embedded SDK Demo</title>
+    <meta charset="UTF-8" />
+    <link
+      rel="stylesheet"
+      type="text/css"
+      href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css"
+    />
+    <link
+      href="https://fonts.googleapis.com/css?family=Heebo&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" type="text/css" href="assets/styles.css" />
+    <link rel="stylesheet" type="text/css" href="assets/toggle.css" />
+  </head>
+
+  <body>
+    <main>
+      <div id="nav">
+        <label
+          >REFRESH CHART&nbsp&nbsp
+          <button id="refresh">⟳</button>
+        </label>
+        <label
+          >REFRESH INTERVAL&nbsp&nbsp
+          <select id="refresh-interval">
+            <option value="0">None</option>
+            <option value="10">10 Seconds</option>
+            <option value="10">1 &nbsp&nbspMinute</option>
+            <option value="100">10 Minutes</option>
+            <option value="100">1 &nbsp&nbspHour</option>
+          </select>
+        </label>
+        <label
+          >FILTER BY COUNTRY&nbsp&nbsp
+          <select id="country-filter">
+            <option value="">--All--</option>
```

Comment:
When you choose this option after selecting another one, you get an invalid filter:
![image](https://user-images.githubusercontent.com/10659986/75512389-627fb000-5a45-11ea-9a24-71fe84db7e95.png)